locations/templates/quarkus-application/template.yaml

```diff
@@ -175,17 +175,32 @@ spec:
             host:
               title: Repo Host
               type: string
-              default: github.com
+              default: gitea.cnoe.localtest.me:8443
```

Comment:
I don't like changing our templates in order to better serve our integration tests. 
It should be the other way around.

Reply:
git provider default reverted to `default: github.com`